specification/machinelearningservices/resource-manager/readme.go.md

```diff
@@ -13,14 +13,14 @@ go:
 
 ```yaml $(go) && $(multiapi)
 batch:
-  - tag: package-2018-03-preview
```

Comment:
This will stop building package-2018-03-preview, is that intentional?  Both can be built.